### PR TITLE
NLP script

### PR DESCRIPTION
**Pull request by nataliemona**

### CODE REVIEW

File: cdk/lib/app.py [hunk 1]
```diff
@@ -9,7 +9,6 @@
 
 stack_props = {
     "site_domain": Param("site_domain").get_value(),
-    "notification_email": Param("notification_email").get_value(),
```
Comment (swt144) [inline cdk/lib/app.py:12]: Think this might have gotten deleted by mistake

File: cdk/lib/service.py [hunk 2]
```diff
@@ -14,43 +14,35 @@ def __init__(
...
-            cloud_map_options=ecs.CloudMapOptions(name="frontend"),
-            desired_count=1,
+            desired_count=desired_count,
             task_definition=task_definition,
             circuit_breaker=ecs.DeploymentCircuitBreaker(rollback=True),
         )
 
         self.service.connections.allow_from_any_ipv4(
-            ec2.Port.tcp(port), "react inbound"
+            ec2.Port.tcp(80), "react inbound"
```
Comment (swt144) [inline cdk/lib/service.py:39]: Possibly updated by mistake to a hardcoded value?

File: cdk/lib/stack.py [hunk 3]
```diff
@@ -13,7 +13,6 @@ def __init__(
         scope: core.Construct,
         id: str,
         site_domain,
-        notification_email,
```
Comment (swt144) [inline cdk/lib/stack.py:16]: Possibly deleted by mistake?

File: etl/etl_target.py [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+from datetime import datetime
+from dotenv import load_dotenv
+from os.path import join, dirname
+
+
+dotenv_path = join(dirname(__file__), ".env")
+load_dotenv(dotenv_path)
+
+
+def connect_and_execute_psql(dbase, query, data):
```
Comment (swt144) [inline etl/etl_target.py]: Don't need to change now, but in the future it might make sense to remove the parameter dbase since it's not used anywhere

**Review by swt144 — COMMENTED** [review]
Have a few comments about properties that may have been deleted by mistake. If that's not the case, I'll approve.

File: etl/etl_target.py (hunk 4 above)
Comment (BlanchardMR) [reply etl/etl_target.py]: 👍

**Review by swt144 — APPROVED** [review]

**Pull request merged**